.github/workflows/release.yml

```
@@ -0,0 +1,33 @@
+name: Release
+
+on:
```

yamllint

[warning] truthy value should be one of [false, true]

(truthy)